docs/monitor-config.md

```diff
@@ -36,7 +36,7 @@ The following config options are common to all monitors:
 | `disableHostDimensions` | `false` | no | `bool` | Some monitors pull metrics from services not running on the same host and should not get the host-specific dimensions set on them (e.g. `host`, `AWSUniqueId`, etc).  Setting this to `true` causes those dimensions to be omitted.  You can disable this globally with the `disableHostDimensions` option on the top level of the config. |
 | `disableEndpointDimensions` | `false` | no | `bool` | This can be set to true if you don't want to include the dimensions that are specific to the endpoint that was discovered by an observer.  This is useful when you have an endpoint whose identity is not particularly important since it acts largely as a proxy or adapter for other metrics. |
 | `dimensionTransformations` |  | no | `map of string` | A map from dimension names emitted by the monitor to the desired dimension name that will be emitted in the datapoint that goes to SignalFx.  This can be useful if you have custom metrics from your applications and want to make the dimensions from a monitor match those. Also can be useful when scraping free-form metrics, say with the `prometheus-exporter` monitor.  Right now, only static key/value transformations are supported.  Note that filtering by dimensions will be done on the *original* dimension name and not the new name. |
-| `extraMetrics` |  | no | `list of string` | Extra metrics to enable besides the default included ones. |
+| `extraMetrics` |  | no | `list of string` | Extra metrics to enable besides the default included ones.  This is an [overridable filter](https://docs.signalfx.com/en/latest/integrations/agent/filtering.html#overridable-filtering). |
 | `extraGroups` |  | no | `list of string` | Extra metric groups to enable besides the metrics that are included by default. |
```

Comment:
Is "metric group" defined somewhere?

Reply:
No, but I added a description of what it is here in this field doc.